database/setup-database.sql

```diff
@@ -11,11 +11,12 @@ CREATE TABLE survey_results (
 CREATE TABLE session (
   id int AUTO_INCREMENT PRIMARY KEY,
   date datetime NOT NULL,
-  location varchar(20) NOT NULL,
+  location_id varchar(5) NOT NULL,
```

Comment:
I think this wants to be an int, not a varchar as it's an index to the location table